src/components/va-text-input/va-text-input.css

```diff
@@ -0,0 +1,58 @@
+:host {
+    display: block;
+    font-family: var(--font-source-sans);
+    margin-top: 3rem;
+}
+
+label {
+    display: block;
+    max-width: 46rem;
+}
+
+span.required {
+    color: var(--color-secondary-dark);
+}
+
+input {
+    appearance: none;
+    border: .1rem solid var(--color-gray);
+    border-radius: 0;
+    box-sizing: border-box;
+    color: var(--color-base);
+    display: block;
+    font-size: 1.6rem;
+    height: 4.2rem;
+    line-height: 1.3;
+    margin: 0.2em 0;
+    max-width: 46rem;
+    padding: 1rem 0.7em;
+    width: 100%;
+}
+
+input:not([disabled]):focus {
+    outline: 2px solid var(--color-gold-light);
+    outline-offset: 2px;
+}
+
+#error-message {
+    color: var(--color-secondary-dark);
+    display: block;
+}
+
+:host([error]) {
+    border-left: 4px solid var(--color-secondary-dark);
+    margin-top: 3rem;
+    padding-bottom: .8rem;
+    padding-left: 1.5rem;
+    padding-top: .8rem;
+    position: relative;
+    right: 1.9rem;
+}
+
+:host([error]) label, .input-error span {
```

Comment:
This `.input-error` seems like a leftover.

Reply:
Dang flabbit. I did a `:%s/\.input-error/:host([error])/` without giving it the `g` on the end to match all occurrences in a line. :roll_eyes: 

Thanks.